setup.py

```diff
@@ -19,7 +19,7 @@
         "service-identity>=14.0.0",
         "treq>=15.0.0",
         "pyOpenSSL>=0.15.1",
-        "txaio==2.1.0",
+        "txaio==2.2.0",
```

Comment:
Ah, interesting. On linux I was getting an error about txaio not having a method called when running it.